operations/deployments/openstad-headless/environments/acc/images.yml

```diff
@@ -1,12 +1,12 @@
 admin:
   deploymentContainer:
-    image: ghcr.io/openstad/admin-server:main-89a9467
+    image: ghcr.io/openstad/admin-server:main-5f286cb
 api:
   deploymentContainer:
-    image: ghcr.io/openstad/api-server:main-89a9467
+    image: ghcr.io/openstad/api-server:main-5f286cb
 auth:
   deploymentContainer:
-    image: ghcr.io/openstad/auth-server:main-89a9467
+    image: ghcr.io/openstad/auth-server:main-5f286cb
 image:
   deploymentContainer:
-    image: ghcr.io/openstad/image-server:main-89a9467
\ No newline at end of file
+    image: ghcr.io/openstad/image-server:main-5f286cb
```

Comment:
Hier moeten we denk ik nog een PR voor maken zodat deze images rechtstreeks vanuit de Github Action worden gevuld, of niet?

Reply:
Uiteindelijk moeten we gaan uitzoeken hoe we dat willen gaan inrichten inderdaad. Kan me best voorstellen dat dat problemen gaat opleveren als bijvoorbeeld bij een PR deze images al gedeployed worden naar een omgeving.

We moeten hier nog iets van een strategie voor bespreken. Voor nu lijkt het mij prima om dat tijdelijk met de hand te doen toch?